copter/source/docs/traditional-helicopter-rsc-setup.rst

```diff
@@ -57,3 +57,10 @@ Turbine Engine Start
 ====================
 
 A special RC Auxiliary Function ("159") is implemented to provide the start signal RC switch for turbine engines. When armed and RSC is idle, the high position signals the helicopter rotor speed governor to ramp the throttle to full and back to idle, which signals the turbine engine ECU to initiate the start sequence. The switch must be set back low and aircraft has to be disarmed to re-enable this feature.
+
+Practice (Manual) Autorotation Setup
+====================================
+
+In versions 4.4 and later, the ability to conduct power re-engagement from an autorotation has been added.  This feature will work with ESC's using their built-in governor, the ardupilot throttle curve or the ardupilot built-in governor.  Three parameters were added to accommodate this feature.
+
```

Comment:
Probably want to remove the last sentence and be consistent with referencing the ESC governor.

Reply:
yeah and need to add links...thanks